### ISSUE TITLE
BetterSqlite3 wrong example

### ISSUE THREAD

**Issue by fecony** [drizzle-team/drizzle-orm-docs#100]
BetterSqlite3 documentation page has code from bun:sqlite docs page example
https://orm.drizzle.team/docs/quick-sqlite/better-sqlite3

```ts
import { drizzle, BunSQLiteDatabase } from 'drizzle-orm/bun-sqlite';
import { Database } from 'bun:sqlite';
```

**Comment by suyogkokaje** [comment]
I would like to fix this issue. Can I work on this?

**Comment by fecony** [comment]
Fixed in #107
